include/astshim/Mapping.h

```diff
@@ -105,16 +104,36 @@ friend class Object;
     bool getReport() const { return getB("Report"); }
 
     /**
-    Get @ref Mapping_TranForward "TranForward": is the forward transform defined?
+    Is the forward transform available?
+
+    Note: this gets the @ref Mapping_TranForward "TranForward" attribute,
```

Comment:
Fair enough. I'll change it though I personally dislike the resulting formatting in Doxygen.